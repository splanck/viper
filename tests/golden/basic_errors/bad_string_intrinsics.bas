10 PRINT LEN(42)
20 LET X$ = "HI"
30 PRINT MID$(X$, "A")
40 END
