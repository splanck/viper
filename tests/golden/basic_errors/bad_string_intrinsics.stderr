B2001(.|\n)*B2001
